github/ci/prow-deploy/files/jobs/kubevirt/cloud-provider-kubevirt/cloud-provider-kubevirt-presubmits.yaml

```diff
@@ -6,9 +6,9 @@ presubmits:
     annotations:
       fork-per-release: "true"
       testgrid-create-test-group: "false"
-    always_run: false
-    optional: true
-    skip_report: true
+    always_run: true
+    optional: false
```

Comment:
nit: you also could just remove this and the following line as they are the default values, but it's also good to have them explicitly, so that everyone can see what the current state is :)